cmd/system-probe/config/config_unsupported.go

```diff
@@ -8,19 +8,18 @@
 package config
 
 import (
-	"fmt"
+	"errors"
 
 	"github.com/DataDog/datadog-agent/pkg/config/model"
 )
 
 const (
-	defaultConfigDir          = ""
-	defaultSystemProbeAddress = ""
+	defaultConfigDir = ""
 )
 
 // ValidateSocketAddress validates that the sysprobe socket config option is of the correct format.
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏You should add a function documentation to explicitly say that it's unsupported on this platform.